### DISCUSSION TITLE
This page could not be found

### DISCUSSION THREAD

**Discussion by oducoelho** [vercel/next.js#45966] [Help] [answered]
Hello guys, today i went open my project to did some things, but when i write in my console "npm run dev", open in my browser one page  with this error: "404: This page could not be found", in my browser console is saying "GET http://localhost:3000/ 404 (Not Found)"

anyone knows how can i fix that.

repository link: https://github.com/oducoelho/space-tourism

**Comment by icyJoseph** [comment, marked as answer]
Hi

A couple of things:

Your project looks like:

```console
.
├── README.md
├── next-env.d.ts
├── next.config.js
├── package-lock.json
├── package.json
├── pages
│   └── api
├── src
│   ├── assets
│   ├── components
│   ├── pages
│   └── styles
└── tsconfig.json

```

Which means you have conflicts because there's `pages` at the top level and `src/pages`.

I'd say first, move `pages/api` into `src/pages/api`:

```console
.
├── README.md
├── next-env.d.ts
├── next.config.js
├── package-lock.json
├── package.json
├── src
│   ├── assets
│   │   ├── crew
│   │   ├── destination
│   │   ├── favicon-32x32.png
│   │   ├── home
│   │   ├── shared
│   │   └── technology
│   ├── components
│   │   └── navbar.tsx
│   ├── pages
│   │   ├── _app.tsx
│   │   ├── _document.tsx
│   │   ├── api
│   │   ├── home.tsx
│   │   └── product.tsx
│   └── styles
│       ├── global.ts
│       ├── index.ts
│       └── pages
└── tsconfig.json
```

Now problem number 2, you don't have a `src/pages/index.tsx` file, which means that `localhost:3000` won't find anything to render.

> Note that at this point `localhost:3000/home` should work already, as well as other pages

Create `src/pages/index.tsx` so now your pages folder looks like:

```console
src/pages
├── _app.tsx
├── _document.tsx
├── api
│   └── hello.ts
├── home.tsx
├── index.tsx
└── product.tsx
```

I guess you meant to do `home.tsx` as `index.tsx`.

And that should be it.

**Reply by oducoelho** [reply]
Thanks men, it's working, god men
